example/serializers.py

```diff
@@ -1,6 +1,12 @@
 from datetime import datetime
+from packaging import version
```

Comment:
`packaging` is a third party package. Could this be moved down a line to be grouped with the other third party packages?